scripts/apps/templates/controllers/CreateTemplateController.ts

```diff
@@ -32,8 +35,27 @@ export function CreateTemplateController(
     this.types = templates.types;
     this.createTypes = _.filter(templates.types, (element) => element._id !== 'kill');
     this.save = save;
+    this.dateTimeFields = null;
 
     activate();
+    function itemData() {
+        const _item = JSON.parse(JSON.stringify(templates.pickItemData(item)));
+
+        self.dateTimeFields?.forEach((field) => {
+            if (_item.extra[field._id]) {
+                Object.values(extensions).forEach(({activationResult}) => {
+                    if (activationResult.contributions?.customFieldTypes) {
+                        activationResult.contributions?.customFieldTypes?.forEach((customField) => {
+                            if (customField?.onTemplateCreate) {
+                                _item.extra[field._id] = customField.onTemplateCreate(field);
```

Comment:
Let's say I had 2 `onTemplateCreate` functions defined. One appends "1" to the value, and other appends "2" to existing value. In the result, it should be both "1" and "2" appended. Current implementation would only use the result of the last iteration.